src/index.js

```diff
@@ -19,7 +19,7 @@ const isProduction = process.env.NODE_ENV === 'production';
 // Base Route Response
 app.get('/', (req, res) => res.json({ status: res.statusCode, message: 'Welcome to Wolfsbane server' }));
 
-app.use(router);
+app.use('/api/v1/', router);
```

Comment:
@sa-ma welldone, is there a reason why you are using this block of code here compared to the previous one??